frontend/packages/dev-console/src/components/pipelines-lists/PipelinesListsPage.tsx

```diff
@@ -10,11 +9,16 @@ import {
 } from '../../models';
 import { Page } from '@console/internal/components/utils';
 import { TechPreviewBadge } from '@console/shared';
+import {
+  MenuAction,
+  MenuActions,
+} from '@console/shared/src/components/multi-tab-list/multi-tab-list-page-types';
+import MultiTabListPage from '@console/shared/src/components/multi-tab-list/MultiTabListPage';
```

Comment:
should import from `@console/shared`
```suggestion
import {
  MenuAction,
  MenuActions,
  MultiTabListPage
} from '@console/shared';
```